server/service/integration_vpp_install_test.go

```diff
@@ -411,6 +390,98 @@ func (s *integrationMDMTestSuite) TestVPPAppInstallVerification() {
 	checkVPPApp(got1, addedApp, installCmdUUID, fleet.SoftwareInstalled)
 	checkVPPApp(got2, errApp, failedCmdUUID, fleet.SoftwareInstallFailed)
 
+	// ================================================
+	// Successful install and delayed verification
+	// ================================================
+
+	t.Run("successful install and delayed verification", func(t *testing.T) {
+		s.fleetCfg.Server.VPPVerifyRequestDelay = 0 * time.Second // Set to 0 to avoid waiting for the verification delay
```

Comment:
So, this is settable but expiration isn't. Curious why there's a difference between the two, and I figure you figured out the answer the hard way.

Reply:
@iansltx just a late night oversight. It's not settable :( 

Removed